frontend/src/scenes/insights/InsightTabs/InsightDisplayConfig.tsx

```diff
@@ -116,7 +116,7 @@ export function InsightDisplayConfig({
 
                 {activeView === ViewType.RETENTION && <RetentionDatePicker />}
 
-                {showFunnelBarOptions && allFilters.display !== FUNNELS_TIME_TO_CONVERT && (
+                {showFunnelBarOptions && allFilters.display === FUNNEL_VIZ && (
```

Comment:
I think this will expose the display layout picker to all funnel viz's, but from prev discussion here (#5094) I think we want to keep this out of time conversion.

Reply:
That hides it from funnel trends and funnel time conversion and should only display for funnel bars, is that in line with what you're saying too?

Reply:
Got it, clarified offline that 

`FUNNEL_VIZ` = bar graph
`FUNNELS_TIME_TO_CONVERT` = histogram
`ACTIONS_LINE_GRAPH_LINEAR` = line graph

Reply:
Hmm, that's the deprecated way. Since we recently added `funnel_viz_type`, it's better to rely on that going forward. In that case, `display` is`FUNNEL_VIZ` for all funnel visualizations, and then `funnel_viz_type` determines if it's `"steps"`, `"trends"` or `"time_to_convert"`.

Reply:
That makes sense, there's actually a helpful comment [here](https://github.com/PostHog/posthog/blob/05e67368ae6176cb276de744227ffa2880482eb5/frontend/src/lib/constants.tsx#L4-L5) that I completely missed that tells us to use `ChartDisplayType` which I think is more in line with what we have on the backend. However I don't today's code has any clear one-to-one mapping of graph types between the backend and frontend. `ChartDisplayType` might be our closest frontend analog to backend's `FunnelVizType`. 

Given today's constraints, I think it makes sense to use `ChartDisplayType` instead of the constants directly.